tests/Runner/FileFilterIteratorTest.php

```diff
@@ -172,7 +172,7 @@ public function testWithoutDispatcher(): void
     public function testInvalidIterator(): void
     {
         $filter = new FileFilterIterator(
-            new \ArrayIterator([__FILE__]),
+            new \ArrayIterator([__FILE__]), // @phpstan-ignore-line
```

Comment:
why?

Reply:
I've added `\Traversable<\SplFileInfo>` type hint to `FileFilterIterator::__construct()`, so PHPStan now detects invalid usage. Probably no need to care about that anyway, as @julienfalque made the same changes in #6468 and just deleted that test.